ngsild-payloads/alert.jsonld

```diff
@@ -0,0 +1,32 @@
+{
+  "id": "urn:ngsi-ld:Alert:7a08a58e-e613-4cc0-9160-8a2636970721",
+  "type": "Alert",
+  "hasSubject": {
+    "type": "Relationship",
+    "object": "urn:ngsi-ld:Event:7a08a58e-e613-4cc0-9160-8a2636970721"
+  },
+  "mailingList": {
+    "type": "Property",
+    "value": {
+      "email": [
+        "user@email.com"
+      ]
+    }
+  },
+  "message": {
+    "type": "Property",
+    "value": {
+      "body": "[body of the mail]",
+      "subject": "[ALERT] Electricity consumption exceeded set value"
+    }
+  },
+  "occurrence": {
+    "type": "Property",
+    "value": true,
+    "observedAt": "2024-04-18T16:40:00Z"
+  },
+  "triggeredBy": {
+    "type": "Property",
+    "value": "urn:ngsi-ld:Subscription:7a08a58e-e613-4cc0-9160-8a2636970721"
+  }
```

Comment:
a relationship is a direct link between : an NGSI-Ld entity, an NGSI-Ld property or another relationship _and_  an object or an unordered array of objects (must be NGSI-Ld entities in both cases). Since a subscription is not an NGSI-Ld entity it cannot be used here